alacritty/src/window.rs

```diff
@@ -313,8 +327,9 @@ impl Window {
     }
 
     #[cfg(not(any(target_os = "macos", windows)))]
-    pub fn set_urgent(&self, is_urgent: bool) {
-        self.window().set_urgent(is_urgent);
+    pub fn set_urgent(&self, _is_urgent: bool) {
+        #[cfg(feature = "x11")]
```

Comment:
These cfgs look very strange. I feel like it should cfg the entire function? Just use the windows `set_urgent` for Wayland?

Reply:
I feel like I've done it like that to not comment call out on caller side.

Reply:
If you merge it with the windows cfg the callsite will not change at all.